docker-compose-beta.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.6'
+services:
+  iris:
+    build: 
+      context: .
+      dockerfile: Dockerfile-amd64
```

Comment:
I was able to build this image successfully but have not been able to create a container using `docker run`. I think the reason lies in the hardcoded `amd64` part here. We should probably get the architecture string from environment variables.